.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: r
+cache: packages
+sudo: false
+r_packages:
+  - readxl
+  - rjson
+  - stringr
+  - org.Hs.eg.db
+  - xlsx
+  - tm
+  - slam
+before_install:
+  - cd code
+script:
+  - Rscript ./AACR2018.R
```

Comment:
this build runs in the data folder?  will generated files go into the correct directory?

Reply:
It run in the code folder, looking at the script it refers to the data as '../data/xxx.yyy'.

Reply:
ok.